### PR TITLE
Add Code of Conduct and improve footer

### PR DESCRIPTION
**Pull request by nelsonmestevao**

### CODE REVIEW

**Comment by alves-luis** [comment]
I think the footer is too tall. There's too much padding between rows

**Comment by nelsonmestevao** [comment]
Using `theme.spacing(1)`:

image

Using `theme.spacing(2)`:
image

Using `theme.spacing(0)`:
image

**Comment by alves-luis** [comment]
In mobile was where it looked more awkward tbf, but yes, I prefer the one with the least spacing

**Comment by nelsonmestevao** [comment]
**Mobile**

Using `theme.spacing(1)`:
image

Using `theme.spacing(2)`:
image

Using `theme.spacing(0)`:
image

Cool?

**Comment by miguelbrandao** [comment]
I like `spacing(0)` better in either version

File: components/Footer.js [hunk 1]
```diff
@@ -1,53 +1,66 @@
...
+  padding: theme.spacing(0),
   display: 'flex',
+  color: 'white',
   justifyContent: 'center',
 });
 
-const StyledAppBar = styled(AppBar)({
-  position: 'absolute',
-  bottom: 0,
+const Wraper = styled(Grid)({
```
Comment (miguelbrandao) [inline components/Footer.js]: It's spelled 'wrapper'

File: pages/codeofconduct.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      pageTitle
+      contentUnderneath
+    />
+    <Document>
+      <Text>
+        This set of rules must be followed throughout the entire event, both at
+        the conference and other conference-related social events.
+      </Text>
+
+      <Title variant="h3">FOR ALL PARTICIPANTS</Title>
```
Comment (miguelbrandao) [inline pages/codeofconduct.js]: `h3` may be a little to big imo...

File: pages/codeofconduct.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  <Layout>
+    <TopSection
+      text="Code of Conduct"
+      color={theme.palette.secondary.main}
+      title
+      pageTitle
+      contentUnderneath
+    />
+    <Document>
+      <Text>
```
Comment (miguelbrandao) [inline pages/codeofconduct.js]: Shouldn't the text be in a JSON and in Portuguese?

Comment (nelsonmestevao) [reply pages/codeofconduct.js]: I don't think json is a good aproach for bigtext. It should be some markdown maybe.

I think english is a better approach to be the most generic.

**Comment by nelsonmestevao** [comment]
What about now? @miguelbrandao

**Pull request closed without merging**